server/opts.go

```diff
@@ -177,6 +177,11 @@ type RemoteLeafOpts struct {
 	tlsConfigOpts *TLSConfigOpts
 }
 
+type JSLimitOpts struct {
+	MaxAckPending int           `json:"-"`
```

Comment:
Don't think we need the json tag for new additions (since Options are not json marshalled)

Reply:
removed